apps/cowswap-frontend/src/modules/swap/hooks/useHandleSwap.ts

```diff
@@ -18,6 +19,7 @@ import { useSwapActionHandlers } from './useSwapState'
 
 export function useHandleSwap(priceImpactParams: PriceImpact): () => Promise<void> {
   const swapFlowContext = useSwapFlowContext()
+  const cowEventEmitter = useCowEventEmitter()
```

Comment:
Add the emitter to the BaseFlowContext so you don't have to pass it individually to each flow

https://github.com/cowprotocol/cowswap/blob/0eabb2049243b24d292a3c12c071fb4a4ec3acdb/apps/cowswap-frontend/src/modules/swap/services/types.ts#L20

Reply:
3a3b230b79a38c3c398531b062d999f857e2f3e7